sdk/platform/src/persistence/mongodb/analytics.rs

```diff
@@ -21,11 +19,10 @@ pub struct Pipeline {
     store: Arc<Store>,
     /// This variable holds all of the Stages to be executed
     /// in the pipeline.
-    stages: Mutex<Vec<Stage>>
+    stages: Mutex<Vec<Stage>>,
```

Comment:
Is this enforced with a linter rule? If not, can it be?

Reply:
See previous comment about linter and splitting large commits.